.github/workflows/test-template.yml

```
@@ -355,7 +355,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        flavor: ['js_pg', 'js_libsql'] # TODO: add more flavors once we want to target SQLite and MySQL support
+        flavor: ['js_pg', 'js_libsql', 'js_d1', 'js_better_sqlite3'] # TODO: add more flavors once we want to target SQLite and MySQL support
```

We already target SQLite, especially with this change. I'd remove this comment altogether, we know we want to support all of our first-class databases.

```suggestion
        flavor: ['js_pg', 'js_libsql', 'js_d1', 'js_better_sqlite3']
```